### PR TITLE
Frontend Master Record Handling

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #3323 

This PR updates the daily docket links on the upcoming dockets page to not point anywhere if the hearing is a master record.

- [ ] Confirm that the master records on the upcoming dockets page don't link through to the daily docket page

### CODE REVIEW

File: client/app/hearings/Dockets.jsx [hunk 1]
```diff
@@ -24,6 +24,20 @@ export class Dockets extends React.Component {
     return index;
   }
 
+  masterRecord = (docket) => {
+    return docket.hearings_array[0].master_record === true;
```
Comment (aroltsch) [inline client/app/hearings/Dockets.jsx]: @sharonwarner So we are assuming that for that one particular day we can have either only master records or non-master records? Is this true in real life?

**Comment by aroltsch** [comment]
I wonder if we should add an additional field to tell a user why the date is not clickable on certain fields?

image

File: client/app/hearings/Dockets.jsx (hunk 1 above)
Comment (sharonwarner) [reply client/app/hearings/Dockets.jsx]: I think so. Doesn't the VACOLS query only pull in a master record if there are no children records for that day?

Comment (aroltsch) [reply client/app/hearings/Dockets.jsx]: Hmm, that's not how it works

Comment (sharonwarner) [reply client/app/hearings/Dockets.jsx]: I emailed Jed to confirm. I'll let you know when he responds!

**Comment by sharonwarner** [comment]
I'll add a new ticket to possibly update the UI so the user knows what's going on. I don't want to hold this ticket up since it's blocking other tickets.

File: client/app/hearings/Dockets.jsx (hunk 1 above)
Comment (sharonwarner) [reply client/app/hearings/Dockets.jsx]: @aroltsch this is good for now!

Comment (aroltsch) [reply client/app/hearings/Dockets.jsx]: Ok! Thanks Sharon!

**Pull request merged**